app/manifest.webapp

```diff
@@ -4,8 +4,10 @@
   "type": "certified",
```

Comment:
nit: should we change app name to "Customizer" , IMO that's visually better than 'Customizer L...'